src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -762,6 +789,7 @@ Copyright (c) .NET Foundation. All rights reserved.
           DependsOnTargets="AssignTargetPaths;
                             DefaultCopyToPublishDirectoryMetadata;
                             _CreateSingleFileHost;
+                            _CreateAppHostForPublish;
```

Comment:
Just curious - does this mean we run the CreateAppHost twice on publish? Once during normal build and then for publish? I also assume the publish one will overwrite the build one on the spot. This might be a double-write which is generally frowned upon during builds...